doc/contribute/committer_expectations.rst

```diff
@@ -0,0 +1,272 @@
+  and there should be tests for the behavior contracts of the API. Maintainers
+  and reviewers have the discretion to determine if the provided tests are
+  sufficient. The examples below demonstrate best practices on how to test APIs
+  effectively.
+
+    - `Kernel timer tests`_ provide better than 85% `test coverage`_ for the
+      kernel timer.
+    - TODO: Provide a driver level API test example.
+
+- Incompatible changes to APIs must also update the release notes for the
+  next release detailing the change.  APIs marked as experimental are excluded
+  from this requirement. [#api-updates]_
+
+- Changes to APIs must increment the API version number according to the API
+  version rules. [#api-version]_
+
+- PRs must also satisfy all :ref:`merge_criteria` before a member of the release
+  engineering team merges the PR into the zephyr tree.
+
+Maintainers may request you break up a PR into smaller PRs and may request you
+create an :ref:`RFC proposal <rfcs>`.
+
+.. _Kernel timer tests: https://github.com/zephyrproject-rtos/zephyr/tree/main/tests/kernel/timer/timer_behavior
+
+.. _test coverage: https://app.codecov.io/gh/zephyrproject-rtos/zephyr/blob/main/kernel/timer.c
+
+Workflow Suggestions That Help Reviewers
+========================================
+
+- Don't resolve comments until the question has been answered or the request
+  completed in an update to the PR.
+
+- Respond to comments from the "Files changed" view. This allows you to respond
+  to multiple comments and publish the responses in bulk. This reduces the
+  number of emails sent to reviewers.
+
+- Try to minimize rebases in the middle of a review. If a rebase is required,
+  push this as a separate update with no functional changes to the PR. When you
+  push a rebase only, add a comment to the PR indicating which commit is the
+  rebase.
+
+- Request re-reviews once comments are addressed. [#re-review]_
+
+PR Review Escalation
+====================
+
+The Zephyr community is a diverse group of individuals, with different levels of
+commitment and priorities. As such, reviewers and maintainers may not get to
+your PR right away.
+
+The `Zephyr Dev Meeting`_ performs a triage of PRs missing reviewer approval.
+This triage process generally follows this process:
+
+#. Identify and update PRs missing an Assignee.
+#. Identify PRs without any comments or reviews, ping the PR Assignee to start a
+   review or assign to a different maintainer.
+#. For PRs that have otherwise stalled, the Zephyr Dev Meeting pings the
+   Assignee and any reviewers that have left comments on the PR.
+
+You may escalate your PR outside of the Zephyr Dev Meeting triage process as
+follows:
+
+- After a few days of inactivity, ping the Assignee or reviewers on your PR by
+  adding a comment to the PR.
+
+- After more than 1 week of inactivity, post a message on the `#pr-help`_
+  channel on Discord linking to your PR.
+
+- After 2 weeks of inactivity, add the `dev-review`_ label to your PR. This
```

Comment:
Note: this requires privileges on GitHub that not all contributors have. Might be worth saying something along the lines of "... or ask someone to do it for you if you can't"